src/Type/WebMozartAssert/AssertTypeSpecifyingExtension.php

```diff
@@ -785,6 +785,12 @@ private function getExpressionResolvers(): array
 				};
 			}
 
+			foreach (['uniqueValues'] as $name) {
```

Comment:
Please write more simple code assigning the key, not a foreach.

Reply:
sure, adapted. sorry, my laziness was on new heights there. it's way simpler now :)